src/config/styles/html.js

```diff
@@ -30,50 +34,64 @@ export const html = {
     marginBottom: normalize(14)
   },
   h3: {
+    margin: 0,
     color: colors.darkText,
     fontFamily: 'bold',
     fontSize: normalize(18),
     lineHeight: normalize(24)
   },
   h4: {
+    margin: 0,
     color: colors.darkText,
     fontFamily: 'regular',
     fontSize: normalize(18),
     fontWeight: '400',
     lineHeight: normalize(24)
   },
   h5: {
+    margin: 0,
     color: colors.darkText,
     fontFamily: 'regular',
     fontSize: normalize(16),
     fontWeight: '400'
   },
   h6: {
+    margin: 0,
     color: colors.darkText,
     fontFamily: 'regular',
     fontSize: normalize(14),
     fontWeight: '400'
   },
   b: {
+    margin: 0,
     fontFamily: 'bold'
   },
   strong: {
+    margin: 0,
     fontFamily: 'bold'
   },
   ul: {
+    margin: 0,
```

Comment:
maybe if margin is 0 now, the `marginBottom: 0` can be removed?

Reply:
yes, can be removed. 👍

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/8705f45ee862bab3878eba4a848868d05380079c